drivers/wireless/ieee80211/bcm43xxx/bcmf_sdio.c

```diff
@@ -718,55 +780,10 @@ int bcmf_bus_sdio_initialize(FAR struct bcmf_dev_s *priv,
       goto exit_free_bus;
     }
 
-  /* Initialize device hardware */
-
-  ret = bcmf_hwinitialize(sbus);
-  if (ret != OK)
-    {
-      goto exit_free_bus;
-    }
-
-  /* Probe device */
-
-  ret = bcmf_probe(sbus);
-  if (ret != OK)
-    {
-      goto exit_uninit_hw;
-    }
-
-  /* Initialize device bus */
-
-  ret = bcmf_businitialize(sbus);
-  if (ret != OK)
-    {
-      goto exit_uninit_hw;
-    }
-
-  nxsig_usleep(100 * 1000);
-
-  sbus->ready = true;
-
-  ret = bcmf_bus_setup_interrupts(sbus);
-  if (ret != OK)
-    {
-      goto exit_uninit_hw;
-    }
-
-  ret = bcmf_sdio_sr_init(sbus);
-  if (ret != OK)
-    {
-      goto exit_uninit_hw;
-    }
-
   /* Register sdio bus */
 
   priv->bus = &sbus->bus;
 
-  /* Start the waitdog timer */
-
-  wd_start(&sbus->waitdog, BCMF_WAITDOG_TIMEOUT_TICK,
```

Comment:
where you call wd_start again?

Reply:
https://github.com/apache/incubator-nuttx/blob/master/drivers/wireless/ieee80211/bcm43xxx/bcmf_sdio.c#L892